vertx-mysql-client/src/main/java/io/vertx/mysqlclient/impl/InitiateSslHandler.java

```diff
@@ -41,7 +41,6 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
 
   @Override
   public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
-    cause.printStackTrace();
```

Comment:
this change will have a merge conflict with your other PR #505, I assume you can just undo this change?

Reply:
no because this PR is for `3.8` branch.